examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -252,11 +253,11 @@
     "ligand_topology = ligand.to_topology()\n",
     "\n",
     "# Load protein and water force field parameters\n",
-    "omm_forcefield = openmm.app.ForceField(\n",
-    "    \"amber99sb.xml\", \"tip3p.xml\"\n",
-    ")\n",
+    "omm_forcefield = openmm.app.ForceField(\"amber99sb.xml\", \"tip3p.xml\")\n",
     "# Teach OpenMM about the ligand molecule and the Parsley force field\n",
-    "smirnoff = SMIRNOFFTemplateGenerator(forcefield=\"openff-1.3.0.offxml\", molecules=[ligand])\n",
+    "smirnoff = SMIRNOFFTemplateGenerator(\n",
+    "    forcefield=\"openff-1.3.0.offxml\", molecules=[ligand]\n",
+    ")\n",
```

Comment:
I think calls to functions should either be a single line:

```python
smirnoff = SMIRNOFFTemplateGenerator(forcefield="openff-1.3.0.offxml", molecules=[ligand])
```

or have each argument on its own line:

```python
smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml", 
    molecules=[ligand],
)
```

But not have multiple arguments per line: 

```python
smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml", molecules=[ligand]
)
```

I think the latter makes it easy to miss arguments other than the first - especially important in examples where users are likely to skim read code. Is it possible to configure black like this? And if that would mean changing the configuration used for the whole project, would that be acceptable?

Reply:
My preference is generally towards opinionated formatters (of which Black is one - "Black is opinionated so you don't have to be") even though I don't personally like all of its defaults (again, including black as a case). That being said, I generally agree with your suggestion here, if only because it makes diffs while adding/removing arguments more legible.

That's all too high-minded, though, since in this case `black`, I think, can be tricked into doing this:

```
$ cat t.py
smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml", molecules=[ligand]
)

smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml", molecules=[ligand],
)

smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml",
    molecules=[ligand],
)
$ black t.py
$ cat t.py
smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml", molecules=[ligand]
)

smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml",
    molecules=[ligand],
)

smirnoff = SMIRNOFFTemplateGenerator(
    forcefield="openff-1.3.0.offxml",
    molecules=[ligand],
)
```

Reply:
Okay, I've gone through the notebooks and tried to find every case of this. I might have missed some, since there is so much to go through, but I think it's all cleaned up now.

Reply:
Took me a second but trailing commas! Nice! Thanks for the tip.